src/util/get-layers.js

```diff
@@ -0,0 +1,26 @@
+
+/**
+ * Extract layers from arc
+ * - finds for `layers` array in json or yaml files
+ * - finds for `layer` tuples in arc file and return a `layers` array
+ */
+module.exports = function getLayers(arc) {
+  if (!arc.aws) return
+
+  // `layers` can come from json or yaml
+  let layers = arc.aws.find(tuple => tuple.includes('layers'))
+  layers = layers && layers[1]
+
+  // `layers` not found, let's check for `layer` tuples in .arc file
```

Comment:
@julianduque I'm unclear on why we're looking for both `layer` and `layers` – is this a holdover from something else, or a check in case of typo? We're assuming a maximum of one runtime, so I'm going to rename to `runtime` per https://github.com/arc-repos/architect/pull/308#issuecomment-469455372, does that sound right?

Reply:
@ryanblock `layers` is parsed when arc config is provided in `json` or `yml` format: https://github.com/arc-repos/arc-parser/pull/8/files

Reply:
Ah, cool, I hadn't seen that, glad you pointed me that way.

Since we're recasting this as `runtime`, would you be cool with me shifting this over to just be a single field for `runtime`? Or do you feel that it's really important to be able to add multiple runtimes? (I can't imagine why but idk!)

Reply:
It can be just a single runtime/layer as you suggested, not a problem at all with that, multiple layer support can come later ;)